src/main/java/org/deepsymmetry/beatlink/AnnouncementListener.java

```diff
@@ -0,0 +1,25 @@
+package org.deepsymmetry.beatlink;
+
+/**
+ * The listener interface for receiving device announcements. This interface is for lower level DeviceAnnouncements
+ * from the AnnouncementSocketConnection class and does not have access to the device list to know if we should send
+ * device found or lost messages. To listen for device found and lost messages, register a DeviceAnnouncementStatusListener
```

Comment:
We’ll want to keep the listener interface class unchanged there as `DeviceAnnouncementListener`, right?

Reply:
And let’s `@link` to any classes/interfaces/methods we mention in the JavaDoc.

Reply:
Yes. And for sure, can do.